.github/instructions/database-rules.instructions.md

```
@@ -0,0 +1,33 @@
+---
+
+# Database Migration Rules
+
+## Migration File Naming
+
+Format: `update-YYYY-MM-DD-description.sql`
+
+## Audit Trail Requirement
+
+Every new table MUST include:
+```sql
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)